gqlalchemy/query_builders/memgraph_query_builder.py

```diff
@@ -79,8 +80,109 @@ def load_csv(self, path: str, header: bool, row: str) -> "DeclarativeBase":
 
         return self
 
+    def construct_subgraph_query(
+        self, node_label: Optional[str] = None, relationship_types: Optional[Union[str, Iterable[str]]] = None
+    ) -> str:
+        """Constructs a MATCH query defining a subgraph using zero or one node labels and relationship types.
+
+        Args:
+            node_label (Optional[str], optional): node label to be used in subgraph
+            relationship_types (Optional[Union[str, Iterable[str]]], optional): types of relationships used in subgraph.
+
+        Returns:
+            str: a MATCH query for a path with given node labels and relationship types
+        """
+        query = " MATCH p="
+        if node_label is None:
+            node = "()"
+        else:
+            node = f"(:{node_label.upper()})"
```

Comment:
```suggestion
node = "()" if node_label is None else f"(:{node_label.upper()})"
```

I am writing this directly here in the comments, so not sure about the correctness. Also, this is just a suggestion, there is nothing wrong with the above. 

Another suggestion is to define the node as a Cypher object somewhere. This could be useful later if you're going to add multiple labels. My suggestion is that this `CypherNode` class is always created like `"(" + [":label" for label in labels]?  + ")"`.  Similar could be applied to directed/undirected relationships.

Reply:
Added this, check it out if its OK

Reply:
Yep, nice :)